entrypoint.sh

```diff
@@ -1,13 +1,23 @@
 #!/bin/bash
 set -e
 
-PG_LOG=/var/log/postgresql/
 PG_CONFIG=/etc/pgbouncer/pgbouncer.ini
 PG_USER=postgres
 
-mkdir -p ${PG_LOG}
-chmod -R 755 ${PG_LOG}
-chown -R ${PG_USER}:${PG_USER} ${PG_LOG}
+# Generate key and certificate for ssl connection
```

Comment:
For docker images, log should only be sent to stdout, creating log file in container is useless and impact performance by writing on aufs layer.